shopping_mall/backend/app/routers/faq.py

```
@@ -602,9 +602,575 @@ def delete_faq_doc(
+    except Exception as e:
+        logger.warning("[policy_articles] ChromaDB 접근 실패: %s", e)
+        return []
+
+    # (chapter, article) → set of clauses
+    article_clauses: dict[tuple[str, str], set[str]] = {}
+
+    for col_name in _ALL_POLICY_COLLECTIONS:
+        try:
+            col = client.get_collection(name=col_name)
+        except Exception:
+            continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)